Ticket/33593

cohosh opened the pull request.

arlolra commented:
The readme still mentions,

```
Proxy:

JavaScript
```

and,

```
cd proxy-go
go build
./proxy-go
```

The `.gitignore` has a bunch unnecessary cruft still.

cohosh commented:
> The readme still mentions,
> 
> ```
> Proxy:
> 
> JavaScript
> ```
> 
> and,
> 
> ```
> cd proxy-go
> go build
> ./proxy-go
> ```
> 
Thanks, fixed in edits to https://github.com/cohosh/snowflake/pull/26/commits/20180dcb041929c1ddd37106e6e74ff8c847f2ee and https://github.com/cohosh/snowflake/pull/26/commits/da01bf232385cd1ae57e818d955990ced478b5df
> The `.gitignore` has a bunch unnecessary cruft still.

Added a commit for this.

I also realized that because the directory name changed, the binary name will now be `proxy` instead of `proxy-go`. I updated the readmes, but we should update the documentation in the wiki: https://trac.torproject.org/projects/tor/wiki/doc/Snowflake#HowtorunaSnowflakeproxy
